css/components/button.css

```diff
@@ -0,0 +1,158 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700;800&display=swap');
+
+
+:root{
+    --lightBlack: #303030;
+    --darkBlack: #000;
+    --white: #fff;
+    --offWhite: #F5F3FF;
+    --navyBlue: #082032;
+    --waterBlue: #003566;
+    --lightWaterBlue: #194975;
+    --lightShadeOfGrey: #e5e5e5;
+    --gray: #9ca3af;
+}
+
+.btn {
+    font-family: "Poppins", sans-serif;
+    font-size: 15px;
+    display: inline-block;
+    border: none;
+    padding: 10px 20px;
+    margin: 5px;
+    border-radius: 5px;
+    cursor: pointer;
+    text-decoration: none;
+}
+
+.btn:focus {
+    outline: none;
+}
+  
+.btn:active {
+    transform: scale(0.97);
+}
+
+.btn-primary{
+    color: var(--white);
+    background-color: var(--waterBlue);
+}
+
+.btn-primary:hover{
+    background-color: var(--lightWaterBlue);
+}
+
+.btn-black{
```

Comment:
A suggestion to avoid using color values in class names. If in case you change the .btn-black to say .btn-violet, then you'll have to change in many places. Better to have a generic name. For e.g. button-default.